src/app/route/CheckoutPage/CheckoutPage.component.js

```diff
@@ -0,0 +1,625 @@
+                                                phone: phone
+                                            }
+                                        }) }
+                                    />
+                                </fieldset>
+
+                                <fieldset block="CheckoutStep" elem="legend">
+                                    <legend>Shipping Method</legend>
+                                    <table block="CheckoutStep" elem="OptionsTable">
+                                        <tbody>
+                                        { Object.keys(shippingMethods).map((index) => {
+                                            return this.renderShippingMethod(index, shippingMethods[index]);
+                                        }) }
+                                        </tbody>
+                                    </table>
+                                </fieldset>
+
+                                <button
+                                    onClick={ () => this.setState({ checkoutStep: CHECKOUT_STEP_REVIEW_AND_PAYMENTS }) }>
+                                    Next step
+                                </button>
+                            </div>
+
+
+                        ) : (
+                            <div block="checkoutStep" elem="reviewAndPayments">
+                                <fieldset block="CheckoutStep" elem="legend">
+                                    <legend>Payment Method</legend>
+                                    <table block="CheckoutStep" elem="OptionsTable">
+                                        <tbody>
+                                        { Object.keys(paymentMethods).map((index) => {
+                                            return this.renderPaymentMethod(index, paymentMethods[index]);
+                                        }) }
+                                        </tbody>
+                                    </table>
+                                </fieldset>
+
+                                <fieldset block="CheckoutStep" elem="legend">
+                                    <legend>Billing Address</legend>
+
+                                    <Field
+                                        id="sameAsShippingAddress"
+                                        type="checkbox"
+                                        block="billingAddress"
+                                        elem="sameAsShippingAddressCheckbox"
+                                        label="My billing and shipping address are the same"
+                                        value={ billingIsSame }
+                                        checked={ billingIsSame }
+                                        onChange={ () => this.setState({ billingIsSame: !billingIsSame }) }
+                                    />
+                                </fieldset>
+
+                                    { billingIsSame ? (
```

Comment:
Move this into separate component. Seems like we might need this in the customer account